core/src/main/scala/com/evolutiongaming/catshelper/FromFuture.scala

```diff
@@ -32,18 +32,19 @@ object FromFuture {
   def summon[F[_]](implicit F: FromFuture[F]): FromFuture[F] = F
 
 
-  implicit def lift[F[_] : Async](implicit executor: ExecutionContext): FromFuture[F] = {
```

Comment:
backward incompatible change, could you please create a new method without ExecutionContext arguments aside reusing a previous?

Reply:
Done. Unfortunately now new factory method cannot be `implicit` in the same scope